### ISSUE TITLE
Admin Registration

### ISSUE THREAD

**Issue by azeng8100** [GTBitsOfGood/VolunTrack#11]
### User Story
* An administrator is an employee at Helping Mamas that sets up events and oversees volunteers. They should have the highest level of access and can grant other users admin privileges.
### Resources and PoC
* https://www.notion.so/gtbitsofgood/f2eff285b3fa4aefa4264ca24d0f77ff?v=17486219c157490e854e5451b2c43879
### Related Tickets
Note for sprint planning: #17 is of lower priority
| Issue Number | Description| Status |
| --- | --- | --- |
| #12 | Send confirmation email | Blocked by #15 |
| #16 | Send reminder email | Blocked by #15 |
| #17 | Manage Waivers | Blocked by #18 |
| #18 | Settings Admin | In Progress |
| #19 | View Signed Up Volunteers | Not Started |
| #21 | Admin Landing Page | Not Started |
